lib/common/constant.go

```diff
@@ -18,4 +18,7 @@ var (
 	// MaxTransactionsInBallot limits the maximum number of `Transaction`s in
 	// one proposed `Ballot`.
 	MaxTransactionsInBallot int = 1000
+	// MaxOperationsInTransaction limits the maximum number of `Operation`s in
+	// one `Transaction`.
+	MaxOperationsInTransaction int = 1000
```

Comment:
That should be `const`

Reply:
@Geod24 `const`, it makes sense, but I want to open to change `MaxOperationsInTransaction` value. I don't know which is better, `var` or `const`.